### ISSUE TITLE
[Feature Request] sort metadata field list on advanced search alphabetically

### ISSUE THREAD

**Issue by jleipsig** [ampache/ampache#2425]
## Describe what you don't like about Ampache, or what it is missing
The list of metadata fields on the advanced search page is sorted on id of the metadata field. An alphabetically sorted list would be nice.

**Comment by lachlan-00** [comment]
i've been playing around with this a bit more and i see that meta fields aren't removed when they're missing from the song anymore.

ie. i removed the tag from a song and it didn't remove from the database unless the song was removed totally.

**Comment by jleipsig** [comment]
Yes, that's already on my list of metadata things. Maybe you prefer to pick this one and/or one of the following?:

1. The select list 'Custom metadata - Disable these fields' on 'preferences.php?action=admin&tab=system' doesn't work. `scub_in` translates it to string 'Array'.
2. The name of preference 'Custom metadata - Define field list' doesn't make clear it's a comma separated list of fields that will be disabled. 
3. Fields defined in `disabled_custom_metadata_fields` and `disabled_custom_metadata_fields_input` are not disabled in advanced search.
4. Sometimes metadata tags are stored under different names in id3v2 and vorbis etc.  (f.e. see https://picard.musicbrainz.org/docs/mappings/). It would be nice to have some sort of tag mapping to store these values in the same field. For example store 'TAG_NAME' as 'Tag Name'. Because ampache is case insensitive, maybe it's better to store all tags in one format (lowercase) and pick a default way to present them (ucfirst/ucwords/uppercase/...) overruled by the case for defined tags in your tag mapping.
